src/components/NormalLeftRow/index.jsx

```diff
@@ -39,13 +39,14 @@ function NormalLeftRow({ classes, treeNode, refType, setSchemaTree }) {
    * If a name property is not defined within the schema,
    * set it to null in order to not display any name.
    */
-  const name = 'name' in schema ? schema.name : null;
+  const name = schema._name || schema.name || null;
```

Comment:
Why not just `schema._name`? Same comment applies to the other places where we do this.

Reply:
Thanks! Applied all changes 👍